src/main/groovy/com/pszymczyk/consul/ConsulStarter.groovy

```diff
@@ -51,7 +53,10 @@ class ConsulStarter {
         }
 
         ConsulPorts consulPorts = new ConsulPorts(httpPort)
```

Comment:
Now we have two config files:

* ports config (required)
* custom config (optional)

I believe we can remove all `ConsulStarterBuilder::withXXXPort()` methods and use config file only or leave all as is...